starter-code/src/functions-and-arrays.js

```diff
@@ -74,6 +160,20 @@ var wordsCount = [
   'disobedience',
   'matter'
 ];
+
+function howManyTimes (palabra1, palabra2) {
+  let contadorPal = 0
+  for (i = 0; i < palabra1.length; i++) {
+    if (palabra1[i] === palabra2){
+        contadorPal++
+    } else {
+        continue
```

Comment:
El `else continue` no es necesario, ya que si no entra en la condición `if`, continúa la siguiente vuelta del bucle `for`